src/features/import/hooks/useColumn.ts

```diff
@@ -42,12 +42,6 @@ export default function useColumn(orgId: number) {
       );
     }
 
-    if (value == 'id') {
-      return !!allSelectedColumns.find(
-        (column) => column.kind == ColumnKind.ID_FIELD
-      );
-    }
-
```

Comment:
At first I thought, what a beautifully simple solution. But then I realized, wait, doesn't this mean you can pick multiple "External IDs", and yes you can. It should be possible to pick one Zetkin ID and one External ID, but no more than that. But right now you can:

![image](https://github.com/user-attachments/assets/f94803a9-1022-4051-88cb-4471aac67906)

(ignore the fact that I've used completely nonsensical data in my screenshot above)

I don't know how best to solve this. I guess we could merge without making any changes to this, but I would ideally like to at least prevent the user from clicking the "VALIDATE" button if there are more than one Zetkin ID and/or more than one External ID.

Would you mind trying to add logic for that? And then maybe create a separate issue for coming up with an even better solution at some later point in time?